src/Services/privateApiService.js

```diff
@@ -0,0 +1,7 @@
+export default function getHeaderAuthorization() {
+  const isTokenSaved = window.localStorage.getItem('token');
+
+  return {
+    Authorization: `Bearer ${isTokenSaved}`,
```

Comment:
what if dont have token in localstorage ?

Reply:
ahi realice una modificación.